libs/ui-toolkit/src/components/accordion/accordion.tsx

```diff
@@ -1,73 +1,71 @@
-import React, { useRef, useState } from 'react';
+import React from 'react';
+import * as AccordionRadix from '@radix-ui/react-accordion';
```

Comment:
Chore (blocking): Matt's started using the 'XPrimitive' naming convention for Radix primitives, so this should become "AccordionPrimitive"